MapboxCoreNavigation/NavigationEventsManager.swift

```diff
@@ -85,9 +85,8 @@ open class NavigationEventsManager: NSObject {
     func start() {
         let eventLoggingEnabled = UserDefaults.standard.bool(forKey: NavigationMetricsDebugLoggingEnabled)
         
-        mobileEventsManager.isDebugLoggingEnabled = eventLoggingEnabled
-        mobileEventsManager.isMetricsEnabledInSimulator = true
```

Comment:
We decided to remove this setting for the same reason expressed in https://github.com/mapbox/mapbox-navigation-ios/pull/304#pullrequestreview-50951199. If navigation events do need to be sent while running in the simulator, we can reenable that workflow in a separate PR.